_posts/2016-10-06-database-testing-with-tsqlt.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+name: database-testing-with-tsql 
+title: Database Testing With tSQLt
+date: 2016-10-06 10:20:00 +00:00
+author: Halima Koundi
+image:
+   src: /assets/img/custom/blog/db-testing/db-testing.jpg

```

Comment:
I need help form Giulia for this one, so I will fix on Monday. As I have broken the aspect ratio of the picture when I tried to resize it. I do not have the right editing tool, Paint is limited :)